src/orientation/fusion/madgwick.c

```diff
@@ -38,31 +38,33 @@ static int zsl_fus_madgwick_imu(struct zsl_vec *g, struct zsl_vec *a,
 	ZSL_VECTOR_DEF(grad, 4);
 	zsl_vec_init(&grad);
 
+
+	// AHRS Convention Variables (https://github.com/Mayitzin/ahrs/)
+	struct zsl_quat q_dot;
+	struct zsl_quat q_gyr;
+
+	q_gyr.r = 0.0;
+	q_gyr.i = g->data[0];
+	q_gyr.j = g->data[1];
+	q_gyr.k = g->data[2];
+
+	// AHRS: qDot = 0.5 * q_prod(q, [0, *gyr]) # (eq. 12)
```

Comment:
For consistency sake, do you mind using /* ... */ for comments?

Reply:
Sure, this is now done